react-ui/src/pages/contest/ContestUnderReview.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * ??? When is this displayed?
```

Comment:
This is displayed when the submission window is closed, but before the voting window is open. This is normally less than a minute, but if there's an issue and we wanted time to resolve it before allowing voting, we could postpone the voting until we are ready.